src/scripts/clipperUI/oneNoteApiUtils.ts

```diff
@@ -70,14 +118,16 @@ export module OneNoteApiUtils {
 		return responseCodeInfo ? responseCodeInfo.isRetryable : false;
 	}
 
+	/**
+	 * Retrieves response code information given that the context is in POSTing a clip.
+	 */
 	function getResponseCodeInformation(apiResponseCode: string): { message: string, isRetryable: boolean, isExpected: boolean } {
 		let handledExtendedResponseCodes = {
 			"10001": { message: Localization.getLocalizedString("WebClipper.Error.GenericError"), isRetryable: true, isExpected: true }, // UnexpectedServerError
 			"10002": { message: Localization.getLocalizedString("WebClipper.Error.GenericError"), isRetryable: true, isExpected: true }, // ServiceUnavailable
 			"10004": { message: Localization.getLocalizedString("WebClipper.Error.PasswordProtected"), isRetryable: false, isExpected: true }, // PasswordProtectedSection
 			"10006": { message: Localization.getLocalizedString("WebClipper.Error.CorruptedSection"), isRetryable: false, isExpected: true }, // CorruptedSection
 			"10007": { message: Localization.getLocalizedString("WebClipper.Error.GenericError"), isRetryable: true, isExpected: true }, // ServerTooBusy
-			"10008": { message: Localization.getLocalizedString("WebClipper.SectionPicker.NotebookLoadUnretryableFailureMessage"), isRetryable: false, isExpected: true }, // SharepointQueryThrottled
```

Comment:
Hmm, I think I'll do a minor cleanup of these methods. Previously, this method held messages for both the clip scenario and GetNotebooks scenario and I wanted to seperate them out so we don't show one message for the other scenario. Perhaps there's a better way of doing things.